model/doctor.go

```diff
@@ -0,0 +1,12 @@
+package model
+
+import (
+	"github.com/go-gorm/gorm"
+)
+
+type Doctor struct {
+	gorm.Model
```

Comment:
Don't use this, you'll struggle with swaggo

Reply:
But our ORM framework can automatically introduce basic CRUD through this. I am doing GO project for the first time and I am not very familiar with it. Sorry.

Reply:
There's no need to be sorry, and I would like to here about the `automatically introduce basic CRUD` part. I don't read any of that, can you provide some information?